packages/test/test-end-to-end-tests/src/test/stashedOps.spec.ts

```diff
@@ -384,12 +430,16 @@ describeNoCompat("stashed ops", (getTestObjectProvider) => {
 			counter.increment(3);
 			const directory = await d.getSharedObject<SharedDirectory>(directoryId);
 			directory.set(testKey, "I will be erased");
+			const string = await d.getSharedObject<SharedString>(stringId);
+			// const collection = string.getIntervalCollection(collectionId);
```

Comment:
Commented out intervalCollection related stuff in just this test for now - should I remove it entirely (and if so, from just this test or the "resends op" test as well)?

Reply:
removing seems fine, as we covered it with other tests